### PR TITLE
[Testnet] Create new testnet chain

### LINKED ISSUE #619: Restart Testnet
- Add large amount of veil mined in the first block
- Change peer version

### PR DESCRIPTION
**Pull request by blondfrogs**
- Added a large amount of veil into the first block mined for regtest, and testnet chains
- Increased the Protocol Version so that only new testnet nodes connect to each other
- Add ability to turn off automint with a console parameter
- Turn on LC_LIMP and POS_WEIGHT BIPs by default of Testnet

Closes #619

### CODE REVIEW

File: src/validation.cpp [hunk 1]
```diff
@@ -2686,9 +2686,13 @@ bool CChainState::ConnectBlock(const CBlock& block, CValidationState& state, CBl
                 FormatMoney(networkReward), FormatMoney(nFounderPayment), FormatMoney(nBudgetPayment),
                 FormatMoney(nLabPayment));
 
-        return state.DoS(100, error("ConnectBlock(): coinbase pays too much (actual=%s vs limit=%s)\n %s",
-                                    FormatMoney(nCreated), FormatMoney(nCreationLimit), block.vtx[0]->ToString()),
-                         REJECT_INVALID, "bad-cb-amount");
+        // On testnet and regtest, we allow the first block to mine lots of coins to make testing easier
+        // We need to make sure this isn't allow on Mainnet.
+        if (Params().NetworkIDString() == CBaseChainParams::MAIN || pindex->nHeight != 1) {
```
Comment (presstab) [inline src/validation.cpp]: Why not just change the reward schedule for testnet instead of this roundabout way?

Comment (blondfrogs) [reply src/validation.cpp]: Great Idea

**Comment by blondfrogs** [comment]
@Fuzzbawls Any reason you can see why this is failing the build checks

**Comment by seanPhill** [comment]
In wallet.cpp line 4787 please change:
`Nothing minted because automint if turned off.` to
`Nothing minted because automint is turned off.`
It gets printed in the debug log periodically (and quite often).

**Comment by blondfrogs** [comment]
There have been lots of new additions to the code base. We should launch a newer testnet than this one. What do you think @presstab

**Comment by blondfrogs** [comment]
Closing due to being outdated. Replaced by #673

**Pull request closed without merging**